watchface

```diff
@@ -23,6 +23,7 @@ Available options:
 -w or --wall WP sets the wallpaper for deploy or test to the named file
 
 Available commands:
+update          use git to update the unofficial-watchfaces project
 version         displays the version of this program and exits
```

Comment:
Grammar inconsistency here. Kinda of a nitpick but i think it might be nice to change all command descriptions to the imperative mood. It does not have a specific tense or subject-verb agreement because it is not referring to a specific subject performing the action.
"displays the version" is a statement describing what happens in the present, while "display the version" is a command or request to do something. Which fits better to describe commands to be executed?
Imho it would be good to have
- use git to ....
- display the version
- push the named
- clone the named

Reply:
Good comment.  Done!